microsoft-graph.d.ts

```diff
@@ -13872,3 +13873,459 @@ export interface TimeRange {
     // End time for the time range.
     endTime?: NullableOption<string>;
 }
+
+
+export namespace CallRecords {
```

Comment:
Inconsistent capitalization between the `microsoftgraph` namespace and `CallRecords` namespace.

Reply:
discussed offline, we agreed on keeping this PascalCase.